samples/connectedhomeip/lock/README.rst

```diff
@@ -104,35 +91,56 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
-Testing the example
-===================
-
-Testing lock state changes
---------------------------
+Testing
+=======
 
 After building the sample and programming it to your development kit, test its basic features by performing the following steps:
 
-#. Observe that initially, **LED 2** is lit which means that the door is closed.
-#. Press **Button 2** to unlock the door. **LED 2** is blinking while the door is opening. After approximately 2 seconds, **LED 2** turns off permanently.
+#. |connect_kit|
+#. |connect_terminal|
+#. Observe that initially **LED 2** is lit, which means that the door lock is closed.
+#. Press **Button 2** to unlock the door.
+   **LED 2** is blinking while the lock is opening.
+   After approximately 2 seconds, **LED 2** turns off permanently.
 #. Press **Button 2** one more time to lock the door again.
-#. Press **Button 1** to initiate factory reset of the device. The device is rebooted after having all its settings erased.
+   **LED 2** starts blinking and then remains turned on.
+#. Press **Button 1** to initiate factory reset of the device.
```

Comment:
@Damian-Nordic , could you add some output lines to these steps, as discussed f2f? Thanks!